src/main/java/tools/map/making/MapCreator.java

```diff
@@ -71,10 +72,15 @@
     return new String[] {TRIPLEA_MAP_FOLDER, TRIPLEA_UNIT_ZOOM, TRIPLEA_UNIT_WIDTH, TRIPLEA_UNIT_HEIGHT};
   }
 
+  /**
+   * Entry point for the map-making utilities application.
+   */
   public static void main(final String[] args) {
-    LookAndFeel.setupLookAndFeel();
+    ClientSetting.initialize();
```

Comment:
@ssoloff It appears a user is having errors with AutoPlacementFinder related to ClientSetting. I see you added initialize to the map maker main method, does this need added to other main methods like: https://github.com/triplea-game/triplea/blob/master/game-core/src/main/java/tools/image/AutoPlacementFinder.java#L54

Reply:
@ron-murhammer Yeah, that's probably the case.  Let me take a look if there are any other considerations.  Otherwise, I'll submit a PR with a `ClientSetting#initialize()` call in every map making tool `main()` method.

Reply:
@ssoloff Yeah, I think for any of those main methods in components of the map creator, `ClientSetting.initialize()` needs called. I can make the PR if you don't have the time. Just wanted to get your thoughts since it appeared you had done something similar for the MapCreator main.

Reply:
@ron-murhammer I just want to verify there's not going to be any problems when `runUtilitiesAsSeperateProcesses` is `false`.  I don't think I checked that before.

Reply:
@ssoloff I actually tried that first to see if it was a work around but setting that to false, also fails for me in the current pre-release but seemingly with a different error. Adding the initilize() call doens't seem to change that failing.

Reply:
It appears to fail before you even get there on this line if you have that set to false https://github.com/triplea-game/triplea/blob/master/game-core/src/main/java/tools/image/AutoPlacementFinder.java#L47

Due to creating components outside the EDT.

Reply:
@ron-murhammer Yeah, it looks like that code is totally broken.  `MapCreator` attempts to call the `main()` methods on a new thread rather than the current EDT.  Might have to fix that in a separate PR.

Reply:
@ssoloff Yeah, I'd say let's just fix having it true for now as I doubt anyone sets the flag to false or they would have reported bugs :)  Can create a separate issue to fix or remove that flag.

Reply:
@ssoloff Did you want to go ahead and submit the PR or want me to? I created an issue for setting runUtilitiesAsSeperateProcesses to false: #3259

Reply:
Working on the PR now.  Thanks for creating the issue. :+1: